packages/vital-elements/doc/ExtendingAndComposing.md

```diff
@@ -1,15 +1,15 @@
 # Extending and Composing Tokens
 
-Vital tokens are designed to be customized.  They provide a starting point
-for the style and behavior of a component, but in almost every case you will
-extend or compose them to meet hte specific requirements of your site,
-reusing what fits your use case, and overriding or extending what does not.
+Vital tokens are designed to be customized.  They provide a starting point for the style and
```

Comment:
Hm - I think we may have diffferent settings on our markdown formatters :).  We should probably agree on a standard.  Here's what I'm using.  Please adopt unless you feel strongly about it:
![image](https://github.com/wodenx/Bodiless-JS/assets/480145/4e81ddcf-4e66-4019-857e-d11cb7ace3cc)
And we should add the plugin suggestion and the default settings tot her epo.

Reply:
tbh i don't care what standard we adopt - so if you prefer whatever method you are using, that's fine - but let's get it into the repo so that everyone is using it.

Reply:
I adopted 100 column-width partially because the ruler is set to 100 in the settings ([`/.vscode/settings.json`](https://github.com/johnsonandjohnson/Bodiless-JS/blob/main/.vscode/settings.json#L34-L36)), and partially because 80 felt too constricting anyway.

- I'd prefer 100 column-width, if you're ok with it.
- I've been using the [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) extension, but I can switch to using [Reflow Markdown](https://marketplace.visualstudio.com/items?itemName=marvhen.reflow-markdown), as you are — I don't think I have a preference there.

Partially related — I've been using these extensions for Markdown, as well, if you think they should be added to the [extension recommendations](https://github.com/johnsonandjohnson/Bodiless-JS/blob/main/.vscode/extensions.json):
- [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
- [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)